HPXMLtoOpenStudio/resources/hpxml.rb

```diff
@@ -615,6 +615,7 @@ class Header < BaseElement
              :software_program_used, :software_program_version, :eri_calculation_version,
              :eri_design, :timestep, :building_id, :event_type, :state_code,
              :begin_month, :begin_day_of_month, :end_month, :end_day_of_month,
+             :dst_enabled, :dst_begin_month, :dst_begin_day_of_month, :dst_end_month, :dst_end_day_of_month,
```

Comment:
Can you rename the existing `begin_month`, etc. variables to `sim_begin_month`, etc.? It will help distinguish from DST variables.